airflow/providers/postgres/provider.yaml

```diff
@@ -53,6 +53,7 @@ dependencies:
   - apache-airflow>=2.4.0
   - apache-airflow-providers-common-sql>=1.3.1
   - psycopg2-binary>=2.8.0
+  - pgvector
```

Comment:
probably should be extra dependency rather than a dependncy?

Reply:
https://github.com/apache/airflow/pull/34891/commits/90dfdba912eaf82795c234a0bae6db2f1e0ac175